src/filesystem/impls/appshell/node/node_modules/fsevents_win/fsevents_win.cpp

```diff
@@ -337,7 +339,7 @@ DWORD WINAPI NodeFSEvents::Run(LPVOID lpData)
 	{
 		// iterate watching for directory changes until we're signalled to stop
 		BOOL bContinue = TRUE;
```

Comment:
According to changes now bContinue reflects status from winapi call at line 346  ReadDirectoryChangesW, in some situations that function may fail and return false according to MSDN, that leads to  looper exit. Do you propose to skip  error handling for that function and continue watching thread as well?